spec/support/node_breadcrumbs_shared_examples.rb

```diff
@@ -0,0 +1,53 @@
+# These are shared examples for breadcrumbs of notes and evidence in a node.
+# For the first argument pass in the action (:new, :edit, :show), and
+# pass the ActiveRecord class (Note, Evidence), as the second argument, e.g.:
+#
+#     include_examples "nodes pages breadcrumbs", :new, Note
+#
+# For the 'show' and 'edit' actions, define a let variable called 'model'
+# which returns the model, e.g.:
+#
+#     let(:model) { @note }
+#     include_examples "nodes pages breadcrumbs", :edit, Note
+#
+# (We can't pass @note in directly, as it doesn't exist in the scope where
+# include_examples is called.)
+
+shared_examples 'nodes pages breadcrumbs' do |action, klass|
+  it 'displays breadcrumbs' do
+    expect(page).to have_selector('.breadcrumb li a', text: 'Nodes')
+    expect(page).to have_selector('.breadcrumb li a', text: @node.label)
+    expect(page).to have_selector('.breadcrumb li a', text: klass.to_s.pluralize)
+  end
+
+  it 'redirects to node page when node label breadcrumb is clicked' do
+    find('.breadcrumb li a', text: @node.label).click
+    expect(page).to have_current_path("/projects/#{current_project.id}/nodes/#{@node.id}")
```

Comment:
Can we use path helpers for this instead of a hard-coded path?

Reply:
Thanks, I've changed it!